nncf/quantization/quantize_model.py

```diff
@@ -223,6 +245,10 @@ def quantize_with_accuracy_control(
     :return: The quantized model.
     :rtype: TModel
     """
+    advanced_quantization_parameters = _update_advanced_quantization_parameters(
```

Comment:
Does `nncf.quantize_with_accuracy_control` support dataset with batch? Do you have an example?
cc` @andrey-churkin

Reply:
ticket # 136183